core/src/main/java/com/microsoft/applicationinsights/TelemetryClient.java

```diff
@@ -215,7 +217,12 @@ private ApplicationInsightsClientImpl createChannel() {
             // TODO (trask) revisit what's an appropriate action here?
             logger.error(e.getMessage(), e);
         }
-
+        // handle AAD authentication
+        // TODO handle authentication exceptions
+        HttpPipelinePolicy authenticationPolicy = AadAuthentication.getInstance().getAuthenticationPolicy();
```

Comment:
won't this fail auth not configured? getInstance() throws exception?

Reply:
Yes if we call init() only when authentication is enabled. for now I removed the auth enabled check to create the authentication object. But we will end up with object with nullable variables.